app/controllers/posts_controller.rb

```diff
@@ -9,4 +9,24 @@ def show
     @user = User.find(params[:user_id])
     @comments = @post.comments
   end
+
+  def new
+    @post = Post.new
+  end
+
+  def create
+    @current_us = current_user
+    @post = current_user.posts.new(params.require(:post).permit(:title, :text))
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** I think it would be better if you use the **"[the strong parameters](https://guides.rubyonrails.org/action_controller_overview.html#strong-parameters)"** like you studied in **[this lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)** for the project requirements